### PR TITLE
Docs- Modified filter pushdown table

### PR DESCRIPTION
**Pull request by mperezfuster**
I have modified the tabled based on the spreadsheet provided.
I have assumed that **Hive (accessing stored as Parquet)** stays the same it was before, can anyone confirm this?

https://mireia-pxf-filter.sc2-04-pcf1-apps.oc.vmware.com/pxf/WIP/filter_push.html

### CODE REVIEW

File: docs/content/filter_push.html.md.erb [hunk 1]
```diff
@@ -41,32 +41,28 @@ PXF filter pushdown can be used with these data types (connector- and profile-sp
...
 | hive (accessing stored as Parquet) | Y, Y<sup>2</sup> | N | N | Y | Y, Y<sup>2</sup> | Y, Y<sup>2</sup> | Y |
-| hive (accessing stored as RCFile or ORC) | Y, Y<sup>2</sup> | N | Y | Y | Y, Y<sup>2</sup> | Y, Y<sup>2</sup> | Y |
+| hive (accessing stored as RCFile) | Y<sup>2</sup> | N | Y | Y | Y, Y<sup>2</sup> | Y, Y<sup>2</sup> | N |
+| hive (accessing stored as ORC) | Y, Y<sup>2</sup> | N | Y | Y | Y, Y<sup>2</sup> | Y, Y<sup>2</sup> | Y |
 | hive:text | Y<sup>2</sup> | N | N | N | Y<sup>2</sup> | Y<sup>2</sup> | N |
-| hive:rc | Y, Y<sup>2</sup> |  N | Y | Y | Y, Y<sup>2</sup> | Y, Y<sup>2</sup> | Y |
+| hive:rc | Y<sup>2</sup> |  N | Y | Y | Y, Y<sup>2</sup> | Y, Y<sup>2</sup> | N |
 | hive:orc | Y, Y<sup>2</sup> |  N | Y | Y | Y, Y<sup>2</sup> | Y, Y<sup>2</sup> | Y |
 | hive:orc and VECTORIZE=true | Y<sup>2</sup> |  N | N | N | Y<sup>2</sup> | Y<sup>2</sup> | N |
 | *:orc (all except hive:orc) | Y<sup>1,3</sup> | N | Y<sup>1,3</sup> | Y<sup>1,3</sup> | Y<sup>1,3</sup> | Y<sup>1,3</sup> | Y<sup>1,3</sup> |
```
Comment (frankgh) [inline docs/content/filter_push.html.md.erb:60]: can we move this line after `*:parquet` above?

**Review by frankgh — APPROVED** [review]

**Comment by denalex** [comment]
@mperezfuster -- the link is no longer functional, is there a new one ?

**Comment by mperezfuster** [comment]
Sorry @denalex I had removed it since Francisco had approved it.
I have created it again, you can check using the same link
https://mireia-pxf-filter.sc2-04-pcf1-apps.oc.vmware.com/pxf/WIP/filter_push.html

File: docs/content/filter_push.html.md.erb [hunk 2]
```diff
@@ -41,32 +41,28 @@ PXF filter pushdown can be used with these data types (connector- and profile-sp
...
+| *:orc (all except hive:orc) | Y<sup>1,3</sup> | N | Y<sup>1,3</sup> | Y<sup>1,3</sup> | Y<sup>1,3</sup> | Y<sup>1,3</sup> | Y<sup>1,3</sup> |
 | s3:parquet and s3:text with S3-Select | Y |  N | Y | Y | Y | Y | Y |
 | hbase | Y | N | Y | N | Y | Y | N |
 | hive (accessing stored as Parquet) | Y, Y<sup>2</sup> | N | N | Y | Y, Y<sup>2</sup> | Y, Y<sup>2</sup> | Y |
-| hive (accessing stored as RCFile or ORC) | Y, Y<sup>2</sup> | N | Y | Y | Y, Y<sup>2</sup> | Y, Y<sup>2</sup> | Y |
+| hive (accessing stored as RCFile) | Y<sup>2</sup> | N | Y | Y | Y, Y<sup>2</sup> | Y, Y<sup>2</sup> | N |
+| hive (accessing stored as ORC) | Y, Y<sup>2</sup> | N | Y | Y | Y, Y<sup>2</sup> | Y, Y<sup>2</sup> | Y |
 | hive:text | Y<sup>2</sup> | N | N | N | Y<sup>2</sup> | Y<sup>2</sup> | N |
-| hive:rc | Y, Y<sup>2</sup> |  N | Y | Y | Y, Y<sup>2</sup> | Y, Y<sup>2</sup> | Y |
+| hive:rc | Y<sup>2</sup> |  N | Y | Y | Y, Y<sup>2</sup> | Y, Y<sup>2</sup> | N |
```
Comment (ashuka24) [inline docs/content/filter_push.html.md.erb:58]: do we need to make a note of the change from y to n here somewhere?

**Review by denalex — APPROVED** [review]

**Pull request merged**